File: 151002/Zavaliuk/Lab1/src/main/java/com/example/rv/impl/tweet/News.java
package com.example.rv.impl.tweet;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class News {
    Long id;
    Long editorId;
    String title;
    String content;
    String created;
    String modified;
}

File: 151001/Novik/DC/src/Publisher/Models/Entity/News.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Publisher.Models.Entity;

public class News
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public long id { get; set; }
    
    [ForeignKey("userId")]
    public long? userId { get; set; }
    
    [Required]
    [StringLength(64, MinimumLength = 2)]
    public String title { get; set; }
    
    [StringLength(2048, MinimumLength = 4)]
    public String? content { get; set; }
    
    public DateTime? created { get; set; }
    
    public DateTime? modified { get; set; }
    
    public User User { get; set; }
    
    public ICollection<Label> Labels = new List<Label>();
    public ICollection<NewsLabel> NewsLabels { get; set; } = new List<NewsLabel>();
}